### DISCUSSION TITLE
Letter Comparison

### DISCUSSION THREAD

**Discussion by imranzaheer612** [IvorySQL/IvorySQL#197] [Q&A]
My ivorysql gives capital 'B' as the result
```
demo=# select greatest('a','b','A','B');''
 greatest 
----------
 B
(1 row)

```

While in the docs,  'b' is shown as the result. [https://www.ivorysql.org/docs/1.5/Compatibillity_Features/conversion_function#examples-14](https://www.ivorysql.org/docs/1.5/Compatibillity_Features/conversion_function#examples-14).

Anyone can help me here?

**Comment by KateElizabethWang** [comment]
Hi, sorry for the late reply. Thank you so much for your question. 
We will give you answer assp.

**Reply by imranzaheer612** [reply]
I guess this problem occurred because of the different collation settings on each pg instances.
